easybuild/easyconfigs/f/Flye/Flye-2.6-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonPackage'
+
+name = 'Flye'
+version = '2.6'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://github.com/fenderglass/Flye'
+description = """Flye is a de novo assembler for long and noisy reads, such as those produced by PacBio
+ and Oxford Nanopore Technologies."""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+
+source_urls = ['http://github.com/fenderglass/Flye/archive/']
+sources = ['%(version)s.tar.gz']
```

Comment:
@boegel Please add the missing checksum.

Reply:
fixed in 0018fc7